File: spring-transaction-test/src/main/java/com/gradle/sample/SampleApplication.java
package com.gradle.sample;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class SampleApplication {

    public static void main(String[] args) {
        //for jenkins test
        SpringApplication.run(SampleApplication.class, args);
    }

}

File: msa-study/sample/src/main/kotlin/com/example/sample/SampleApplication.kt
package com.example.sample

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker
import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.context.annotation.Bean
import org.springframework.web.client.RestTemplate

@SpringBootApplication
@EnableCircuitBreaker
@EnableEurekaClient
class SampleApplication

fun main(args: Array<String>) {
    runApplication<SampleApplication>(*args)
}